SPECS/rust/CVE-2024-43806.patch

```diff
@@ -0,0 +1,1596 @@
+From c822bb4e0f93b377e53996f241aaa6cc9d1fb3e6 Mon Sep 17 00:00:00 2001
```

Comment:
Which commit is this? I am taking a look at the github [advisory](https://github.com/advisories/GHSA-c827-hfw6-qwvm) and I see 4 similar commits there which only change 3 files. I see this patch has more changes and so wanted to ask

Reply:
Great question. I've expanded on this in the description, but the patch file combines the four patches needed for the four different versions of `rustix` in the vendored code for `rust`, along with the updated checksums.